templates.d/nginx.conf.txt

```diff
@@ -692,6 +692,13 @@ http {
   # origin headers not debugged
   %%ENDIF
 
+  %%BEGIN
+  # Set custom log status
+  log_format compression '$remote_user [$time_local] %DNS_DOMAIN% '
```

Comment:
Heya! I am working on a different but equivalent solution. Question: why capture $remote_user?

Reply:
Sorry, no reason. That was a mistake.